src/CONST.js

```diff
@@ -296,6 +297,7 @@ const CONST = {
     GOOGLE_MEET_URL_ANDROID: 'https://meet.google.com',
     DEEPLINK_BASE_URL: 'new-expensify://',
     PDF_VIEWER_URL: '/pdf/web/viewer.html',
+    CLOUDFRONT_REGEX_STRING: `^https://\\w+.${CLOUDFRONT_DOMAIN}`,
```

Comment:
1. Let's make this a regex and not a string.
2. Change the const name to `CLOUDFRONT_DOMAIN_REGEX`.
3. The regex should be case insensitive.
4. Make sure it does not pass with cases like  https://notcloudfront.net.